### PR TITLE
Fix CI to tf==1.15, add compatibility note to README

### PR DESCRIPTION
**Pull request by blazekadam**

### CODE REVIEW

**Comment by blazekadam** [comment]
Please spare me a lecture on how we should properly deal with that issue, either take it or leave it.

File: .circleci/config.yml [hunk 1]
```diff
@@ -8,14 +8,16 @@ references:
       command: |
         apt-get update -qy
         apt-get install -y git python3-dev python3-pip curl
-        pip3 install coveralls coverage tensorflow
+        pip3 install coveralls coverage tensorflow==1.14
```
Comment (FloopCZ) [inline .circleci/config.yml:11]:
```suggestion
        pip3 install coveralls coverage tensorflow==1.15
```

**Comment by petrbel** [comment]
Nice! Have thinking about this for a while. I suggest fixing the version in other repos as well.

File: .circleci/config.yml (hunk 1 above)
Comment (blazekadam) [reply .circleci/config.yml:11]: `1.15` does not work there, it is not within the available versions, do not ask me why

**Comment by blazekadam** [comment]
okay, never mind then

File: .circleci/config.yml (hunk 1 above)
Comment (petrbel) [reply .circleci/config.yml:11]: btw this won't work with new python 3.8... we have to move to virtualenv also in ci

**Pull request closed without merging**